### ISSUE TITLE
Message dialog callback after clicking Delete btn not called

### ISSUE THREAD

**Issue by amedranogil** [universAAL/ui#143]
_Originally Opened: Ioannis Moulos (2012-05-23 11:47:43_)
_Originally Closed: 2012-05-24 15:13:43_

Using bus.ui/1.1.0 and dm.mobile/1.1.0
Message dialog callback after clicking Delete btn not called.

	public void dialogAborted(String dialogID) {
		System.out.println(&quot;-----DIALOG ABORTED--------&quot;); 
//THIS IS NOT SHOWN WHEN I CLICK DELETE ON THE MESSAGE DIALOG
	}
	 private Form initMainDialog() {
			Form f = Form.newMessage(&quot;MPLA MPLA&quot;,&quot; message&quot;);
			return f;
		    }
	 
	 void startMainDialog() {
			if (mainDialog == null)
			    mainDialog = initMainDialog();
			UIRequest out = new UIRequest(new Resource(), mainDialog,
				LevelRating.middle, Locale.ENGLISH, PrivacyLevel.insensible);
			sendUIRequest(out);
		    }

--

From: _this issue has been automatically imported from our old issue tracker_

**Comment by amedranogil** [comment]
_Original author: @amedranogil (2012-05-24 15:13:43)_

Messages don't react in the ui.bus. 
Messages are intended for one way communication, AAL applications send messages to the user to inform about something, and user can either delete the message or keep it to read it afterwards. Think of messages as an email service where applications can send &quot;emails&quot; to the user, but the user cannot send emails back; there is no return receipt (the ui framework ensures the user will receive the notification though).

Bottom line applications cannot recover if the messages have been deleted or not.

**Comment by amedranogil** [comment]
issue cloed on 2012-05-24 15:13:43
